tools/build_defs/native_tools/BUILD

```diff
@@ -1,68 +1,65 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
-load("//for_workspace:cmake_build.bzl", "cmake_tool")
-load("//for_workspace:make_build.bzl", "make_tool")
-load("//for_workspace:ninja_build.bzl", "ninja_tool")
-load(":native_tools_toolchain.bzl", "native_tool_toolchain")
 
-native_tool_toolchain(
+_DEPRECATION_NOTICE = "This target has moved to `@rules_foreign_cc//toolchains/...`"
+
+alias(
     name = "preinstalled_make",
-    path = "make",
+    actual = "//toolchains:preinstalled_make",
+    deprecation = _DEPRECATION_NOTICE,
+    tags = ["manual"],
     visibility = ["//visibility:public"],
 )
 
-make_tool(
+alias(
     name = "make_tool",
-    make_srcs = "@make//:all",
+    actual = "//toolchains:make_tool",
+    deprecation = _DEPRECATION_NOTICE,
     tags = ["manual"],
+    visibility = ["//visibility:public"],
 )
 
-native_tool_toolchain(
+alias(
     name = "built_make",
-    path = "make/bin/make",
-    target = ":make_tool",
+    actual = "//toolchains:built_make",
+    deprecation = _DEPRECATION_NOTICE,
+    tags = ["manual"],
     visibility = ["//visibility:public"],
 )
 
-native_tool_toolchain(
+alias(
     name = "preinstalled_cmake",
-    path = "cmake",
-    visibility = ["//visibility:public"],
-)
-
-cmake_tool(
-    name = "cmake_tool",
-    cmake_srcs = "@cmake//:all",
+    actual = "//toolchains:preinstalled_cmake",
+    deprecation = _DEPRECATION_NOTICE,
     tags = ["manual"],
+    visibility = ["//visibility:public"],
 )
 
-native_tool_toolchain(
+alias(
     name = "built_cmake",
-    path = "cmake/bin/cmake",
-    target = ":cmake_tool",
+    actual = "//toolchains:built_cmake",
+    deprecation = _DEPRECATION_NOTICE,
+    tags = ["manual"],
     visibility = ["//visibility:public"],
 )
```

Comment:
Happy to leave these here but I don't think anyone can be using these as I don't think the `built_*` targets for toolchains actually work.

Reply:
I'll remove this after the release